packages/speech/src/index.js

```diff
@@ -87,6 +88,9 @@ function Speech(options) {
   };
 
   common.GrpcService.call(this, config, options);
+  this.api = {
+    speechApi: v1beta1(options).speechApi()
```

Comment:
Renamed to `Speech`, to match the spelling and capitalization of the actual gRPC service name. With this convention, `this.api` will actually look quite similar to `this.protos`.